brozzler/chrome.py

```diff
@@ -222,6 +222,7 @@ def start(
             chrome_args.append("--ignore-certificate-errors")
         if proxy:
             chrome_args.append("--proxy-server=%s" % proxy)
+        self.logger.info(f"Chrome launched with args {chrome_args} proxy is {proxy}")
```

Comment:
Is this intended to be a temporary debug statement before this is merged, or permanent? If the latter, you can probably just combine this with the logging statement two lines down by adding `proxy=proxy` to the logger dictionary.

Reply:
It was useful in debugging a failing test. Probably worth keeping, so I'll add this to the other log line.